src/components/config-tabs/denominators/OrgUnitLevelSelect.js

```diff
@@ -0,0 +1,57 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { SingleSelectFieldFF, ReactFinalForm } from '@dhis2/ui'
+import React, { useMemo } from 'react'
+
+const { Field } = ReactFinalForm
+
+const ORG_UNITS_LEVELS_QUERY = {
+    orgUnitLevels: {
+        resource: 'organisationUnitLevels',
+        params: {
+            paging: false,
+            fields: ['id', 'displayName', 'level'],
+            order: 'level:asc',
+        },
+    },
+}
+
+export const OrgUnitLevelSelect = () => {
+    const { loading, error, data } = useDataQuery(ORG_UNITS_LEVELS_QUERY)
+
+    const dataElementGroupOptions = useMemo(
+        () =>
+            data
+                ? data.orgUnitLevels.organisationUnitLevels.map(
+                      ({ level, displayName }) => ({
+                          label: displayName,
+                          value: level.toString(),
+                      })
+                  )
+                : null,
+        [data]
+    )
+
+    const placeholderText = useMemo(() => {
+        if (loading) {
+            return 'Loading...'
+        }
+        if (error) {
+            return 'An error occurred'
+        }
+        return 'Select an org unit level'
+    }, [loading, error])
+
+    return (
+        <div>
+            <Field
```

Comment:
@mauricejulesm : one other thing I realized (which I noticed only after trying something for external relations work), the value for the minimum level is expected to be a number, but you are saving a string, which causes some issues when the level gets interpreted (e.g. we check if a level is mathematically lower/higher the selected org unit, but this doesn't work with a string)

Can you add a parse/format to convert to number on this field (like this:
https://github.com/hisprwanda/who-data-quality-annual-report/blob/main/src/components/config-tabs/numerator-relations/EditNumeratorRelationModal.js#L137-L138)